sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/administration/models/PersonalizerPolicyResultTotalSummary.java

```diff
@@ -1,12 +1,219 @@
+    private Float ipsEstimatorNumerator;
+
+    /*
+     * Timestamp of the aggregation.
+     */
+    private OffsetDateTime timeStamp;
+
+    /**
+     * Creates an instance of PersonalizerPolicyResultTotalSummary class.
+     */
+    PersonalizerPolicyResultTotalSummary() {
+    }
+
+    /**
+     * Get the averageReward property: Average reward.
+     *
+     * @return the averageReward value.
+     */
+    @Override
+    public Float getAverageReward() {
+        return this.averageReward;
+    }
+
+    /**
+     * Get the confidenceInterval property: Gaussian confidence interval for the Policy evaluation.
+     *
+     * @return the confidenceInterval value.
+     */
+    @Override
+    public Float getConfidenceInterval() {
+        return this.confidenceInterval;
+    }
+
+    /**
+     * Get the sumOfSquares property: Sum of Squares for the Policy evaluation results.
+     *
+     * @return the sumOfSquares value.
+     */
+    @Override
+    public Float getSumOfSquares() {
+        return this.sumOfSquares;
+    }
+
+    /**
+     * Get the aggregateTimeWindow property: Time window for aggregation.
+     * For example, PT5M (5 mins). For information about the time format,
+     * see http://en.wikipedia.org/wiki/ISO_8601#Durations.
+     *
+     * @return the aggregateTimeWindow value.
+     */
+    @Override
+    public Duration getAggregateTimeWindow() {
+        return this.aggregateTimeWindow;
+    }
+
+    /**
+     * Get the snipsEstimatorDenominator property: Denominator for SNIPS estimator.
+     *
+     * @return the snipsEstimatorDenominator value.
+     */
+    @Override
+    public Float getSnipsEstimatorDenominator() {
+        return this.snipsEstimatorDenominator;
+    }
+
+    /**
+     * Get the ipsEstimatorDenominator property: Denominator for IPS estimator.
+     *
+     * @return the ipsEstimatorDenominator value.
+     */
+    @Override
+    public Float getIpsEstimatorDenominator() {
+        return this.ipsEstimatorDenominator;
+    }
+
+    /**
+     * Get the ipsEstimatorNumerator property: Numerator for IPS estimator.
+     *
+     * @return the ipsEstimatorNumerator value.
+     */
+    @Override
+    public Float getIpsEstimatorNumerator() {
+        return this.ipsEstimatorNumerator;
+    }
+
+    /**
+     * Get the timeStamp property: Timestamp of the aggregation.
+     *
+     * @return the timeStamp value.
+     */
+    @Override
+    public OffsetDateTime getTimeStamp() {
+        return this.timeStamp;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PersonalizerPolicyResultTotalSummary setNonZeroProbability(Float nonZeroProbability) {
+        super.setNonZeroProbability(nonZeroProbability);
+        return this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        jsonWriter.writeStartObject();
+        jsonWriter.writeNumberField("nonZeroProbability", getNonZeroProbability());
```

Comment:
Does this look right? We are just writing one property here but the type has several other properties.

Reply:
Same answer as [below](https://github.com/Azure/azure-sdk-for-java/pull/42503#discussion_r1811377749)